README.md

```diff
@@ -32,9 +32,42 @@ descopeClient := client.New()
 descopeClient := client.NewWithConfig(&client.Config{ProjectID: projectID})
 ```
 
-## Usage
+## Authentication Functions
 
-Here are some examples how to manage and authenticate users:
+These sections show how to use the SDK to perform various authentication/authorization functions:
+
+1. [OTP Authentication](#otp-authentication)
+2. [Magic Link](#magic-link)
+3. [Enchanted Link](#enchanted-link)
+4. [OAuth](#oauth)
+5. [SSO/SAML](#ssosaml)
+6. [TOTP Authentication](#totp-authentication)
+7. [Passwords](#passwords)
+8. [Session Validation](#session-validation)
+9. [Roles & Permission Validation](#roles--permission-validation)
+10. [Logging Out](#logging-out)
+
+## API Managment Function
```

Comment:
Why `API Management Function` and not only `Management Functions`?

Reply:
That's fair, I will change the headers though for these sections then to match the table of contents: 
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/32936811/236547324-ebd6524b-f231-4b16-a902-0df5b3e1c4ca.png">